pkgs/by-name/va/vanillatd/package.nix

```diff
@@ -1,23 +1,30 @@
 {
   lib,
   stdenv,
+  appName,
```

Comment:
`appName` is also a special config for this package, so it should be put with `CMAKE_BUILD_TYPE`